openstreetmap-carto.style

```diff
@@ -70,17 +147,22 @@ node,way   waterway     text         polygon
 node,way   wetland      text         polygon
 node,way   width        text         linear
 node,way   wood         text         linear
-node,way   z_order      int4         linear # This is calculated during import
-way        way_area     real                # This is calculated during import
+way        way_area     real         linear # This is calculated during import
+
+# Columns defined in openstreetmap-carto.lua file
+way        z_order      int4         linear
+way        osmcarto_z_order int4     linear
 
 # Area tags
```

Comment:
We can get rid of this line and everything below if we're using a Lua transform.